companies/management/commands/importregistrations.py

```diff
@@ -0,0 +1,140 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.files.base import ContentFile
+from django.db import transaction
+from django.utils.text import slugify
+from companies.models import Company, Contact
+from people.models import Programme as Program
+from exhibitors.models import Exhibitor, CatalogInfo, \
+        WorkField, JobType, Continent, Value
+from fair.models import Fair
+from urllib.request import urlopen
+from urllib.error import HTTPError
+from ast import literal_eval
+import csv
+
+
+anmalan_url = 'http://anmalan.armada.nu'
```

Comment:
If that's what's recommended by PEP then maybe. I don't like it though since all variables that are not being mutated (most of them) should be declared constants if you had a programming language that actually had true support for constants. Making the vast majority of the variables in your code scream using capital letters look ugly and makes it seem like constants are a weird abnormal thing, when in reality they should be the default way to declare variables unless you mutate them.